packages/manager/cypress/e2e/domains/smoke-domain-import-a-zone.spec.ts

```diff
@@ -0,0 +1,95 @@
+import { ImportZonePayload } from '@linode/api-v4/types';
+import { domainFactory } from '@src/factories';
+import { authenticate } from 'support/api/authentication';
+import { fbltClick } from 'support/helpers';
+import { randomDomainName, randomIp } from 'support/util/random';
+import { mockGetDomains, mockImportDomain } from 'support/intercepts/domains';
+import { ui } from 'support/ui';
+
+authenticate();
+describe('Import a Zone', () => {
+  /*
+   * - Clicks "Import A Zone" button and confirms operation.
+   * - Confirms that Domain won't be imported when the domain is empty or invalid.
+   * - Confirms that Domain won't be imported when the name server is empty or invalid.
+   * - Confirms that Domain exists after imported operation.
+   */
+  it('imports a zone in the domain page', () => {
+    const zone: ImportZonePayload = {
+      domain: randomDomainName(),
+      remote_nameserver: randomIp(),
+    };
+
+    const mockDomain = domainFactory.build({
+      domain: zone.domain,
+      group: 'test-group',
+    });
+
```

Comment:
```suggestion

    mockGetDomains([]).as('getDomains');
    cy.visitWithLogin('/domains');
    cy.wait('@getDomains');

    ui.button
      .findByTitle('Import a Zone')
      .should('be.visible')
      .should('be.enabled')
      .click();
  
```

Looks like the initial `cy.visitWithLogin()` and interactions may have slipped through the cracks during refactoring, etc.! Adding this allows these to pass as expected.